simplerisk-ubuntu1804/docker-compose.yml

```diff
@@ -15,36 +19,3 @@ services:
     ports:
       - "8080:80"
       - "8443:443"
-    depends_on:
-      - database
```

Comment:
I dont know if we can get rid of this as they have obviously designed it to have a database but instead maybe optional? what do you think?

Reply:
Good question. Not sure if you can have a service depend on an optional service? I'll have to RTFM, but maybe a separate docker-compose.yml might be better? Not sure what best practice would look like... 🤔